sdk/metric/selector/simple/simple.go

```diff
@@ -70,7 +69,7 @@ func (selectorInexpensive) AggregatorFor(descriptor *sdkapi.Descriptor, aggPtrs
 	case sdkapi.GaugeObserverInstrumentKind:
 		lastValueAggs(aggPtrs)
 	case sdkapi.HistogramInstrumentKind:
-		aggs := minmaxsumcount.New(len(aggPtrs), descriptor)
+		aggs := sum.New(len(aggPtrs))
```

Comment:
Is this selector behavior specified somewhere? It might make sense to add a comment here if so linking to the specification.

Reply:
I don't think so. It seems like the `selectorInexpensive` selector aims to keep in faster and uses less memory than the others, which uses `minmaxsumcount` aggregators for `Histogram` instruments. I'm not sure if this selector should remain or remove. Any suggestions?

Reply:
This mechanism will be replaced, probably also the associated `AggregatorFor` interface could change or be refactored, as Views are implemented. As far as the original intent, having the "inexpensive" support in a Histogram instrument means a zero-bucket histogram. We're waiting for a future OTLP that supports optional fields to do that, at which point MinMaxSumCount can return in the form of a zero-bucket histogram with min and max fields.

Reply:
A quick inspection of the current Histogram aggregator shows that it will support zero buckets. If you want to change this to Histogram with boundaries set to `nil`, that would be IMO a better way to replace `MinMaxSumCount`. However, I don't think the zero-bucket configuration is tested and I'd like to see that if we go this way.